pages/k8s/release-notes.md

```diff
@@ -60,4 +82,4 @@ Please see [this page][historic] for release notes of earlier versions.
 [haoverview]:
 [metallb-docs]:
 [hacluster-docs]:
-
+[dns-provider config description]: https://github.com/juju-solutions/kubernetes/blob/5f4868af82705a0636680a38d7f3ea760d35dadb/cluster/juju/layers/kubernetes-master/config.yaml#L58-L67
```

Comment:
This config description should probably be brought into kubernetes-docs somewhere, but I wasn't quite sure where it should go. New page probably, maybe **Operations -> Addons** or **Reference -> Charm configuration**?

Reply:
It should. A charm-configuration set of pages is planned, but for now I think it makes perfect sense for this to go in the 'ugrade-notes' pages, linked to from here. I will add those changes to this PR so you can see.